src/go/intelligence/int_test/test_wrapper.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/sh
+##H Script for automation of testing process of the intelligence module.
+##H Usage: test_wrapper.sh <work_directory>
+##H
+
+# If user provided the input parameter
+if [ $# -ne 1 ]; then
+    perl -ne '/^##H/ && do { s/^##H ?//; print }' <$0
+    exit 1
+fi
+
+# Setup work directory based on user input
+WDIR=$1
+TOP=$(dirname $1)
+
+if [ ! -d $WDIR ]; then
+    echo "Provided work directory '$WDIR' does not exists. creating..."
+
+    if [ ! -w $TOP ]; then
+        echo "can not create $WDIR: $USER has no write permission for '$TOP' directory. exiting."
+        exit 1
+    fi
+
+    if mkdir -p $WDIR; then
+        echo "$WDIR created !"
+    else
+        echo "Unable to create $WDIR"
+        exit 1
+    fi
+fi
+
+#Variables
+AM_BIN=$WDIR/am/alertmanager
+AM_CONFIG=$WDIR/am/alertmanager.yml
+TEST_CONFIG=$WDIR/test_config.json
+AM_VERSION="alertmanager-0.21.0.linux-amd64"
+AM_URL="https://github.com/prometheus/alertmanager/releases/download/v0.21.0/${AM_VERSION}.tar.gz"
+CMSMONITORING_REPO_URL="https://github.com/indrarahul2013/CMSMonitoring.git"
```

Comment:
this is wrong since you point to your repo, while it should points to official repo, i.e. github.com/dmwm/CMSMonitoring

Reply:
fixed this.